packages/postgrest/lib/src/postgrest_query_builder.dart

```diff
@@ -94,6 +94,8 @@ class PostgrestQueryBuilder<T> extends PostgrestBuilder<T, T> {
   ///
   /// By default no data is returned. Use a trailing `select` to return data.
   ///
+  /// [defaultToNull] Make missing fields default to `null`. Otherwise, use the default value for the column.
```

Comment:
Could we update the comments to something like this to make it clear that it applies to only when inserting multiple rows in bulk?

```suggestion
  /// When inserting multiple rows in bulk, [defaultToNull] can be used to set the values of the missing fields to be either `null` or the default value for the column.
```

But even this is still kind of unclear. This new behavior is a bit tricky, isn't it 😂

Reply:
Great idea! I've added another sentence for single row insertions in https://github.com/supabase/supabase-flutter/pull/550/commits/8686868f02f273fd105b66a323e53b9a7edf323a